### PR TITLE
[IMP] qweb: document 't-cache' and 't-nocache' directives

### PR DESCRIPTION
**Pull request by Gorash**
related with: https://github.com/odoo/odoo/pull/88276

### CODE REVIEW

**Review by Julien00859 — APPROVED** [review]
Review done verbally, we know there are still rooms for improvement but this it the best we have right now and we are [under pressure](https://youtu.be/a01QQZyl-_I) to merge.

File: content/developer/reference/frontend/qweb.rst [hunk 1]
```diff
@@ -546,6 +773,10 @@ At a deeper level than the previous helper is the ``render`` method on
                        used to expand or customize QWeb locally (by creating
                        a "new" qweb based on ``ir.qweb`` with alterations)
 
+.. py:method:: render(template_name, values, load, **options)
+    :func:`load(ref)`
+        returns etree object, ref
```
Comment (AntoineVDV) [inline content/developer/reference/frontend/qweb.rst:778]: A blank line is required in order for the method to render correctly.
```suggestion
.. py:method:: render(template_name, values, load, **options)

    :func:`load(ref)`
        returns etree object, ref
```

**Review by AntoineVDV — CHANGES_REQUESTED** [review]
Since this PR is blocking the freeze, I only reviewed its technical aspects and didn't pay much attention to the content itself. The latter would however need to be proofread someday to fix the grammar and phrasing, eventually on the occasion of a full rewrite of the page.

@robodoo delegate+

While you're at it, could you please update the message commit to be relevant to the documentation repository? Something like: `[IMP] qweb: document 't-cache' and 't-nocache' directives`.

**Comment by AntoineVDV** [comment]
@robodoo r+

**Comment by AntoineVDV** [comment]
@robodoo squash

**Comment by Gorash** [comment]
robodoo r+

**Pull request closed without merging**